src/common/cockpitmemfdread.c

```diff
@@ -48,6 +54,9 @@ cockpit_memfd_read (int      fd,
       return NULL;
     }
 
+workaround:
+  ;
```

Comment:
why the weird line with just a `;` here?

Reply:
You can't have a variable declaration following a label, so this turns it into a statement.